CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
+- Added basic SEO for better google discoverability - [#283](https://github.com/orgs/DigitalExcellence/projects/11#card-45582851)
```

Comment:
Changelog entry should link to issue page, not the issue card in the project overview 😃